retrieval_service/requirements.txt

```diff
@@ -1,7 +1,7 @@
 asyncpg==0.29.0
 fastapi==0.109.2
 google-auth==2.32.0
-google-cloud-firestore==2.14.0
+google-cloud-firestore @ git+https://github.com/googleapis/python-firestore.git@2de16209409c9d9ba41d3444400e6a39ee1b2936
```

Comment:
The library is now released https://github.com/googleapis/python-firestore/releases. This can be updated

Reply:
That release has vector similarity search but not async vector similarity search. Releases had been stopped since there is a recent breach  https://github.com/googleapis/python-firestore/pull/933

Reply:
This is now released https://pypi.org/project/google-cloud-firestore/2.17.0/

Reply:
I still see a git link here?